File: problemas/1000/1010.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let total = 0.0;

for (let i = 0; i < 2; i++) {
  const [, quantidade, valor] = entrada.shift().split(' ');
  total += quantidade * valor;
}

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);

File: problemas/1000/1010.dart
import 'dart:io';

void main() {
  double total = 0.0;

  for (var i = 0; i < 2; i++) {
    List entrada = stdin.readLineSync().split(' ');
    int quantidade = int.parse(entrada[1]);
    double valor = double.parse(entrada[2]);
    total += quantidade * valor;
  }

  stdout.writeln('VALOR A PAGAR: R\$ ${total.toStringAsFixed(2)}');
}